### DISCUSSION TITLE
Firma elettronica del dichiarante

### DISCUSSION THREAD

**Discussion by franthemanIT** [italia/ansc#1062] [Supporto Webapp]
salve, al paragrafo 26 della guida operativa si dice dice piu' o meno: "Se il dichiarante ha la firma digitale, entra con SPID e prende visione".
Domanda che sorge spontanea: a che gli serve la firma digitale?
Domande a margine ma comunque importanti:
- perche' con SPID un meccanismo di presa visione e non la "firma SPID" ex art. 20 del CAD con tanto di linee guida e specifiche tecniche AgID?
- come è possibile che il dichiarante usi la sua firma digitale? si scarica il pdf come per la firma analogica ma si ricarica poi con firma digitale?

**Comment by franthemanIT** [comment]
Sulla terza domanda, leggendo fra le righe la [specifica del servizio cooperativo “R006_firma_dichiarante”](https://github.com/italia/ansc/blob/main/docs/openapi/R006_firma_dichiarante.yaml) (che descriverebbe, a prima vista, solo l’upload del filo firmato esternamente) e la [tabella di decodifica ANSC_10_dec_tipo_file](https://github.com/italia/ansc/blob/main/docs/Decodifiche/10_dec_tipo_file.csv) sembra di capire che, in teoria, l'atto firmato dal dichiarante possa essere caricato anche in p7m.
Anche se la tabella ANSC_10 dovrebbe valere anche per altri upload, per i quali nella guida si specifica esplicitamente la possibilità che siano nativi digitali.

**Comment by acolosimosogei** [comment]
Salve, 

la sua segnalazione è stata presa in carico,
Le daremo un risconto quanto prima.

Saluti, 
Assistenza ANSC

**Comment by asqueglia** [comment]
Buon pomeriggio, la firma elettronica del dichiarante non è una firma remota ma una presa visione dell'anteprima dell'atto; questa si rende possibile con l'invio da parte del sistema ANSC di una email al dichiarante; questa email contiene un link a cui si accede necessariamente tramite browser ad esempio dal proprio dispositivo mobile; la pagina mostrata richiederà un accesso tramite credenziali digitali spid cns o cie; dopo l'autenticazione il dichiarante vedrà l'anteprima dell'atto e potrà prenderne visione con l'apposito pulsante 'Conferma'. Quiesta è la procedura della firma elettronica del cittadino. Il tutto è spiegato nella guida operativa ANSC. Chiudiamo pertanto la discussion.
